lib/elixir/lib/record.ex

```diff
@@ -193,6 +193,9 @@ defmodule Record do
   In the example above, a set of macros named `user` but with different
   arities will be defined to manipulate the underlying record.
 
+      # We must import the module first
```

Comment:
Maybe "We must import the module first in order to make the `user` macros available"? Would it be too much? Also the point is not really making them available, but callable without `User.`. 😕 My fear is that like this it's not clear *why* we would import the module, but maybe it's overthinking it :)